deploy.sh

```diff
@@ -103,16 +103,13 @@ function terraform_exec {
 }
 
 function terraform_cleanup {
-  # Terraform doesn't seem to remove node pool taints without recreating the
-  # node pool, whereas this approach causes less downtime.
-  pool=(base-pool "--cluster=cloud-robotics" "--zone=${GCP_ZONE}" "--project=${GCP_PROJECT_ID}")
```

Comment:
Do we not need these lines anymore?

Reply:
Sorry, forgot to mention in the commit message. This is an old cleanup from when we first enabled Workload Identity last year.